WcaOnRails/spec/controllers/competitions_controller_spec.rb

```diff
@@ -49,6 +49,15 @@
         expect(response).to render_template :new
       end
     end
+
+    context 'when signed in as a regular user' do
+      sign_in { FactoryGirl.create :user }
+
+      it 'doesn\'t allow access' do
```

Comment:
How about `does not` instead of `doesn't`? Or use double quotes instead of single quotes for the string.